riva/proto/riva_nmt.proto

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+//
+// NVIDIA CORPORATION and its licensors retain all intellectual property
+// and proprietary rights in and to this software, related documentation
+// and any modifications thereto.  Any use, reproduction, disclosure or
+// distribution of this software and related documentation without an express
+// license agreement from NVIDIA CORPORATION is strictly prohibited.
+
+syntax="proto3";
+
+package nvidia.riva.nmt;
+
+option cc_enable_arenas = true;
+option go_package = "nvidia.com/riva_speech";
+
+/*
+ *  RivaTranslation service provides rpcs to translate between languages.
+ *
+ */
+service RivaTranslation {
+
+  // Translate text to text, from a source to a target language.  Currently source language is required, along with the model name.
+  // Multiple texts may be passed per request up to the given batch size for the model, which is set at translation pipeline creation time.
+rpc TranslateText(TranslateTextRequest) returns (TranslateTextResponse) {}
+
+  // Lists the available language pairs and models names to be used for TranslateText
+  rpc ListSupportedLanguagePairs(AvailableLanguageRequest) returns (AvailableLanguageResponse) {}
+}
+
+// request for synchronous translation of each text in texts.
+// Available languages can be queries using ListSupportLanguagePairs RPC.
+// source and target languages must be specified, are currently two character ISO codes, this will likely change to BCP-47 inline with other Riva Services for GA.
+//
+message TranslateTextRequest {
+    repeated string texts = 1;
+    string model = 2;
+    string source_language = 3;
+    string target_language = 4;
+}
+
+// contains a single translation, collecting into the translate text response
+// Includes the language code, since with multi lingual models there are multiple possibilities.
+message Translation {
+    string text = 1;
+    string language = 2;
+}
+
+// Translations are returned as text:language pairs.  These are 1:1 for the passed in 'texts' from the request.
+message TranslateTextResponse {
+    repeated Translation translations = 1;
+}
+
+// by default returns all available languages.
+// can specificy an individual model as well.
+message AvailableLanguageRequest {
+    // If empty returns all available languages.
+    string model = 1;
```

Comment:
can you rename the model field to model_and_language  to be more consistent with the comments ?

Reply:
I dont follow, this is the string to specify the model?  Is the comment unclear?

Reply:
// by default returns all available languages.
// can specificy an individual model as well.                                                                                                                                   
Based on message name "AvailableLanguageRequest" and above comments, it is unclear if the field name "model" is to specify only models or languages as well.

Reply:
updated the comment.  Keep in mind this proto has already shipped, and is being simply pushed so unable to change fields.